frontend/public/sample_returning_members.csv

```diff
@@ -0,0 +1 @@
+NetID,Graduation Semester,Returning
```

Comment:
The returning member survey doesn't include Graduating Semester as a field, can we fetch this from our backend instead?